asv_bench/asv.conf.json

```diff
@@ -0,0 +1,103 @@
+{
+    // The version of the config file format.  Do not change, unless
```

Comment:
The github syntax highlighting is going nuts on this - is this a variation of JSON that has comments?

Reply:
I guess?  I just used the pandas version as a template.